src/components/organisms/DashboardItems.tsx

```diff
@@ -6,7 +6,7 @@ import { ParentSize } from '@visx/responsive';
 import DataWrapper from '../molecules/DataWrapper';
 import { useDispatch, useSelector } from 'react-redux';
 import { RootState } from '../../redux';
-import visualisationMapping, { VisualisationMappingFunctionPath } from '../../utils/visualisationMapping';
+// import visualisationMapping, { VisualisationMappingFunctionPath } from '../../utils/visualisationMapping';
```

Comment:
Hey, it looks like you have commented this out? Are there any other ways to remove this import?